uint8 position_request
uint8 speed_request
uint8 force_request
---
bool success
